### PR TITLE
Adding site map for testing purposes

### PR DESCRIPTION
**Pull request by jcmagsay**
### What's this PR do?
- Make landing page default when you land on `/`
- Add exact match routing
- Add Site Map page to help with qa and dev 
- Update page hero text for Create Account page
- Add Link to nav logo

### What are the relevant tickets?
https://mondorobot.atlassian.net/browse/RET-281

### Who should review this PR?
@jcmagsay @mhfen

### CODE REVIEW

File: src/client/components/logos/logo-nav.jsx [hunk 1]
```diff
@@ -1,34 +1,36 @@
...
+    <Link to="/">
+      <svg width="145" height="24" viewBox="0 0 145 24" aria-labelledby={ICON_ID}>
+        <title id={ICON_ID}>Logo Nav</title>
```
Comment (mhfen) [inline src/client/components/logos/logo-nav.jsx]: I would either drop this aria-labelledby scheme for this particular svg and go with a visually hidden span inside the <Link> OR update this to say "Home Page". I think I favor the first option as less confusing to the user?

File: src/client/components/logos/logo-nav.jsx [hunk 2]
```diff
@@ -1,34 +1,36 @@
...
+    <Link to="/">
```
Comment (mhfen) [inline src/client/components/logos/logo-nav.jsx]: This probably needs block or inline-block level display for legacy browsers.

File: src/client/pages/site-map.jsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React, {Component} from 'react';
+import {Link} from 'react-router-dom';
+import MakeHash from 'make-hash';
+import PageHero from 'components/heroes/page-hero';
+import routes from 'routes/routes';
+import Text from 'components/txt';
+
+export default class SiteMap extends Component {
```
Comment (mhfen) [inline src/client/pages/site-map.jsx]: Is this what they meant by Site Map, or is this more of a `sitemap.xml` requirement for search indexing?

File: src/client/components/logos/logo-nav.jsx (hunk 1 above)
Comment (jcmagsay) [inline src/client/components/logos/logo-nav.jsx]: sure thing, totally agree with option 1

File: src/client/pages/site-map.jsx (hunk 3 above)
Comment (jcmagsay) [inline src/client/pages/site-map.jsx]: This is so Todd and I can test the site on mobile without having to type in all the routes with hyphens. We'll comment out this page and the route when we hand off to Retul

File: src/client/components/logos/logo-nav.jsx (hunk 2 above)
Comment (jcmagsay) [inline src/client/components/logos/logo-nav.jsx]: done

File: src/client/pages/site-map.jsx (hunk 3 above)
Comment (mhfen) [inline src/client/pages/site-map.jsx]: 👍

**Pull request merged**